recipe/build-lib.sh

```diff
@@ -1,5 +1,20 @@
 CUDA_CONFIG_ARG=""
 if [ ${cuda_compiler_version} != "None" ]; then
+    # for documentation see
+    # docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#building-for-maximum-compatibility
+    # docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#major-components__table-cuda-toolkit-driver-versions
+    # docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list
+
+    # default of `./configure --with-cuda-arch=...` corresponds to compute_{35,52,60,61,70,75}
+    ARCHES=(35 52 60 61 70)
+    if [ ${cuda_compiler_version} != "9.2" ]; then
+        # cuda 9.2 does not support Turing (sm_75)
+        ARCHES+=(75)
+    fi
+    for arch in "${ARCHES[@]}"; do
+        CUDA_ARCH="${CUDA_ARCH} -gencode=arch=compute_${arch},code=compute_${arch}";
+    done
```

Comment:
after this line, in order to support PTX JITing, add another gencode along the lines of:
`CUDA_ARCH="${CUDA_ARCH} -gencode arch=compute_${last_arch},code=compute_${last_arch}"`
where `last_arch` is the last element in the `ARCHES` array.

Reply:
Thanks for the feedback - pushed a change!